src/main/java/uk/gov/pay/adminusers/resources/ServiceResource.java

```diff
@@ -152,10 +152,18 @@ public Response updateServiceAttribute(@PathParam("serviceExternalId") String se
         LOGGER.info("Service PATCH request - [ {} ]", serviceExternalId);
         return serviceRequestValidator.validateUpdateAttributeRequest(payload)
                 .map(errors -> Response.status(BAD_REQUEST).entity(errors).build())
-                .orElseGet(() -> serviceServicesFactory.serviceUpdater().doUpdate(serviceExternalId, ServiceUpdateRequest.from(payload))
-                        .map(service -> Response.status(OK).entity(service).build())
-                        .orElseGet(() -> Response.status(NOT_FOUND).build()));
+                .orElseGet(() -> processPayload(serviceExternalId, payload));
+    }
 
+    private Response processPayload(String serviceExternalId, JsonNode payload) {
+        try {
+            final List<ServiceUpdateRequest> requests = ServiceUpdateRequest.getUpdateRequests(payload);
+            return serviceServicesFactory.serviceUpdater().doUpdate(serviceExternalId, requests)
+                    .map(service -> Response.status(OK).entity(service).build())
+                    .orElseGet(() -> Response.status(NOT_FOUND).build());
+        } catch (ValidationException e) {
```

Comment:
It seems the convention for other endpoints in this resource is to just let the exception propagate upwards